.appveyor.yml

```diff
@@ -5,7 +5,10 @@ platform:
   - x86
   - x64
 install:
+  - vcpkg integrate install
   - cd %APPVEYOR_BUILD_FOLDER%
   - git submodule update --init --recursive
+  - vcpkg install gflags:x86-windows-static
+  - vcpkg install gflags:x64-windows-static
```

Comment:
You can probably make the install more specific to the build (x86 vs x64) using the `PLATFORM` environment variable. That would made cache rebuilding faster after detecting a broken cache, and also keep the cache files smaller. The two configurations will be cached separately, so there should be no conflict between the configurations over what files to cache.

Perhaps even better, is to put the install steps in a batch file so they can be run locally.

For a batch file, I recommend by default to install both x86 and x64. That would be most convenient for local development, where the user may freely switch configurations and rebuild. For CI builds, the batch file can look for the presence of the `PLATFORM` environment variable, and if found install only for that platform.

For an example, you can check the NAS2D project:
https://github.com/lairworks/nas2d-core/blob/master/.appveyor.yml
https://github.com/lairworks/nas2d-core/blob/master/InstallVcpkgDeps.bat

You can also make the caching depend on the batch file, that way the cache is automatically broken and rebuilt when you updated dependencies in the batch file.